backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -230,6 +234,15 @@ private boolean duplicateAttachmentExist(Project project) {
         return false;
     }
 
+    private boolean updateProjectAllowed(Project project, User user) {
+        if (project.clearingState != null && project.clearingState.equals(ProjectClearingState.CLOSED)
+                && !(PermissionUtils.isUserAtLeast(UserGroup.ADMIN, user)
```

Comment:
```
PermissionUtils.isUserAtLeast(UserGroup.ADMIN, user) || PermissionUtils.isUserAtLeast(UserGroup.SW360_ADMIN, user)
```
is exactly the same as only
```
PermissionUtils.isUserAtLeast(UserGroup.SW360_ADMIN, user)
```
since `UserGroup.ADMIN > UserGroup.SW360_ADMIN` it is <pre>PermissionUtils.isUser<b>AtLeast</b></pre>

Reply:
Thanks.Corrected the same.